third_party/libusb/naclport/src/chrome_usb/types.cc

```diff
@@ -212,6 +263,20 @@ void EndpointDescriptorUsageConverter::VisitCorrespondingPairs(
   callback(EndpointDescriptorUsage::kNotification, "notification");
 }
 
+template <>
+StructValueDescriptor<EndpointDescriptor>::Description
+StructValueDescriptor<EndpointDescriptor>::GetDescription() {
+  return Describe("chrome_usb::EndpointDescriptor")
+      .WithField(&EndpointDescriptor::address, "address")
+      .WithField(&EndpointDescriptor::type, "type")
+      .WithField(&EndpointDescriptor::direction, "direction")
+      .WithField(&EndpointDescriptor::maximum_packet_size, "maximumPacketSize")
+      .WithField(&EndpointDescriptor::synchronization, "synchronization")
+      .WithField(&EndpointDescriptor::usage, "usage")
+      .WithField(&EndpointDescriptor::polling_interval, "pollingInterval")
+      .WithField(&EndpointDescriptor::extra_data, "extra_data");
```

Comment:
nit: `extraData` (I see the same is done in some VisitFields methods, it may be good to change it there too)

Reply:
Ack - This naming is coming from the chrome.usb API: https://developer.chrome.com/apps/usb#type-EndpointDescriptor
We have to use the same here, otherwise we'll not be able to convert this object coming from the Javascript-based Extensions API into a C++ struct (and vice versa - the API will raise an error if we call a method with a struct with a wrongly named field).

The .h file, which admittedly isn't visible in the PR, mentions that the structs are mimicking the Extensions API.

Reply:
As per offline discussion, I've added a comment into every relevant `GetDescription()` function that explains where these strings are coming from.